code/game/objects/items/devices/chameleonproj.dm

```diff
@@ -64,6 +64,8 @@
 
 /obj/item/device/chameleon/proc/disrupt(delete_dummy = 1)
 	if(active_dummy)
+		for(var/mob/M in active_dummy)
+			M << "<span class='danger'>Your chameleon-projector deactivates.</span>"

```

Comment:
this isn't a list
just do
`active_dummy << "<span class='danger'>Your chameleon-projector deactivates.</span>"`